package.json

```diff
@@ -4,24 +4,24 @@
   "author": "Devon Harris",
   "license": "MIT",
   "scripts": {
-    "dev": "webpack-dev-server --progress --colors",
+    "dev": "concurrently --kill-others \"yarn run watch\" \"yarn start\"",
```

Comment:
Found `concurrently` in a past tutorial. It allows running two commands simultaneously so you don't need two terminals anymore.